### ISSUE TITLE
ModelGridColumn with ModelCombo

### ISSUE THREAD

**Issue by cbsistem** [altsoft/PlatypusJS#69]
Example:

    var modelCombo = new ModelCombo();
         modelCombo.displayList = model.owners;
         modelCombo.displayField = 'ownerName';

    var g = new ModelGrid();
          g.data = model.allcustomers;
         var colService = new ServiceGridColumn();
         g.addColumnNode(colService);
         var colName = new ModelGridColumn();
             colName.field = 'name';
             colName.title = 'just added name column'

linking?   colName  with  modelCombo

    g.addColumnNode(colName);
    form.view.add(g, {left: '300px', top: '10px', right: '9px', bottom: '48px'});

**Comment by marat-gainullin** [comment]
Assign column's property 'editor' to 'ModelCombo' instance just created.
Additionally, combo columns have to be presentation of references, not scalars as 'name'. It should be 'colName.field = "owner";' (if "owner" is property of 'model.customers' array elements). Platypus.js ORM is able to automatically add reference properties based on foreign keys between model's entities. Note, that foreign key is one property in entity's data array and reference property based on it is another property. Names of reference properties is settled up with designer via a context menu of corresponding dash line between entities in module's model view.

**Comment by marat-gainullin** [comment]
Reference properties can be created by ORM as well as with your JavaScript code with wide variety of ways...

**Comment by cbsistem** [comment]
in example PetHotel, OwnerView, no exist datamodel,  model is passed by reference in runtime. (DATAMODEL is empty) ;
for this reason. it is not possible to change Model GridColumn grdPets.petType to petTypesQuery

When I fill out ModelCombo.displayList property in ModelGridColumn grdPets.petType, this column does not list the petTypesQuery model. Does not accept that I write manually using the IDE Designer.

Solution:
1 in OwnerView.Datamodel, ADD petTypesQuery
2 chage ModelGridColumn grdPets.petType, ModelCombo.displayList to petTypesQuery
3 Save
4-1- in OwnerView.Datamodel, REMOVE petTypesQuery
5-SAVE

**Comment by marat-gainullin** [comment]
Can you clarify please what for do you change grdPets.petType.editor.displayList from data array of OwnersView's data model to another data array from OwnerView's data model?

**Comment by marat-gainullin** [comment]
Regarding your solution. It is quite well, but PetHotel example is organized so to avoid bunch of network requests (one for each `OwnerView` module instance). Pet types is commonly used data and so, it can be pulled from server only once.

**Comment by cbsistem** [comment]
I did not change.
For some reason, grd Pets.pet Type.editor.de playlist was empty.
So I tried to replace and was not getting.

**Comment by marat-gainullin** [comment]
Oh. It is another story... :) Can you please describe the problem from the beginning? Do examples work properly on site and on your computer. What is the problem and when does it appear?

**Comment by marat-gainullin** [comment]
I've double checked just now examples work properly both on site and in fresh example project (new project -> Examples -> Platypus.js -> PetHotel) created via designer on local machine. There is data in `form.grdPets.petType.editor.displayList` in both cases.

**Comment by jskonst** [comment]
Could you please share us example project (github, archive, or some another way). Append to the project database (h2 database) and credentials - username and password. Using this materials we will be able to solve problems of you project.
